### ISSUE TITLE
Options in functions

### ISSUE THREAD

**Issue by Jamonster123** [SkriptLang/Skript#5701]
### Skript/Server Version

```markdown
Server Version: git-Paper-372 (MC: 1.18.2)
Skript Version: 2.6.4
Installed Skript Addons: 
 - SkQuery v4.1.9
 - SkBee v2.9.0 (https://github.com/ShaneBeee/SkBee)
 - Khoryl v1.0.6
Installed dependencies: 
 - WorldGuard v7.0.7+216b061
```

### Bug Description

I have some options and I'm trying to have them as a function default so like `func: int = {@option}`. It works just fine when reloading just that script but when I reload all scripts, it gives the error 'undefined option {@range_default}'.

### Expected Behavior

No errors should be given just like when reloading the single script.

### Steps to Reproduce

Make a new script, and paste the following: 

options:
	function_option: 10

function testfunction(stuff: integer = {@function_option}):
	broadcast {@function_option}

command /test:
	trigger:
		testfunction()

You can use the command after reloading just the script (not all scripts) to see that it does in fact work. No idea why but github doesn't want me to use code and won't show the tabs so you'll have to add those yourself.

### Errors or Screenshots

![replace](https://github.com/SkriptLang/Skript/assets/71259585/a443bee8-aaf0-4e8b-bf20-29ca37b0c0f0)

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by TheLimeGlass** [comment]
Can confirm the issue is present in 2.6.4 on 1.18.2 but not on 2.7.0-beta2 on 1.19.

It appears to have been fixed by some of the structure changes done in that time.

The issue is the regex grabbing the name. For some reason just because it's called `function_option` I believe Skript is trying to parse it as a function even though there is an underscore there. When I changed the name to `functionsss_option` I could not replicate the issue. So try and frame from using the prefix `function_` in your options to avoid this issue.
